_examples/golang-sse/webapp/src/rpc.ts

```diff
@@ -0,0 +1,477 @@
+/* eslint-disable */
```

Comment:
can you rename this file to `rpc.gen.ts` I prefer to use `.gen.xxx` for filenames whenever code is generated so its obvious